src/main/resources/assets/openblocks/lang/de_de.lang

```diff
@@ -393,8 +393,8 @@ itemGroup.tabOpenBlocks=OpenBlocks
 entity.openblocks.luggage.name=Reisegepäck
 entity.openblocks.cartographer.name=Kartograph
 entity.openblocks.block.name=Schwebender Block
-entity.openblocks.mini_me.name=Mini-Me
+entity.openblocks.mini_me.name=Kleines Ich
```

Comment:
This is a direct reference and probably shouldn't be translated.

Reply:
I didn't know.